rosco-core/src/main/groovy/com/netflix/spinnaker/rosco/api/Artifact.groovy

```diff
@@ -28,4 +28,7 @@ class Artifact {
   String version
   String reference
   Map<String, String> metadata
+  String auth
```

Comment:
This should be using the kork-artifacts package instead of recreating this here

Reply:
Cool, I hadn't seen the `kork-artifacts`. Fixing